homeassistant/components/switch/isy994.py

```diff
@@ -0,0 +1,82 @@
+""" Support for ISY994 lights. """
+# system imports
+import logging
+
+# homeassistant imports
+from homeassistant.components.isy994 import (ISY, ISYDeviceABC, SENSOR_STRING,
+                                             HIDDEN_STRING)
+from homeassistant.const import STATE_ON, STATE_OFF  # STATE_OPEN, STATE_CLOSED
+# The frontend doesn't seem to fully support the open and closed states yet.
+# Once it does, the HA.doors programs should report open and closed instead of
+# off and on. It appears that on should be open and off should be closed.
```

Comment:
Yes. Specifically, my garage door. Right now it is an on/off switch. On is open, Off is closed. The controller takes care of the rest. Maybe one day, I might clone the switch state cards on the frontend to make door cards. I'll need some help getting into that.